Source/Core/Core/WiiRoot.cpp

```diff
@@ -32,6 +34,48 @@ namespace FS = IOS::HLE::FS;
 
 static std::string s_temp_wii_root;
 
+static std::string FileNameFromPath(const std::string& path)
```

Comment:
Is it worth putting this in StringUtil? (Also, it's a shame we still cannot use std::filesystem because of Android :/)

Reply:
Done.